src/keysignature.ts

```diff
@@ -32,27 +33,36 @@ export class KeySignature extends StaveModifier {
 
     this.setKeySig(keySpec, cancelKeySpec, alterKeySpec);
     this.setPosition(StaveModifierPosition.BEGIN);
-    this.glyphs = [];
+    this.children = [];
     this.paddingForced = false;
   }
 
   // Add an accidental glyph to the `KeySignature` instance which represents
   // the provided `acc`. If `nextAcc` is also provided, the appropriate
   // spacing will be included in the glyph's position
-  convertToGlyph(acc: { type: string; line: number }, nextAcc: { type: string; line: number }, stave: Stave): void {
+  protected convertToGlyph(
+    acc: { type: string; line: number },
+    nextAcc: { type: string; line: number },
+    stave: Stave
+  ): void {
     const code = Tables.accidentalCodes(acc.type);
     const glyph = new Element(Category.KeySignature);
     glyph.setText(code);
 
     // Determine spacing between current accidental and the next accidental
-    const extraWidth = 1;
+    let extraWidth = 1;
     // Place the glyph on the stave
     glyph.setYShift(stave.getYForLine(acc.line));
-    if (this.glyphs.length > 0) {
-      const prevGlyph = this.glyphs[this.glyphs.length - 1];
+    if (this.children.length > 0) {
+      const prevGlyph = this.children[this.children.length - 1];
+      if (
+        (prevGlyph.getText() === Glyphs.accidentalNatural || glyph.getText() === Glyphs.accidentalNatural) &&
+        Math.abs(glyph.getYShift() - prevGlyph.getYShift()) < 10
+      )
+        extraWidth = 2;
```

Comment:
The indentation looks weird here.

Is it because you didn't add braces { }?

We should always use braces because it improves readability and reduces bugs. Maybe we should add that rule to the eslint, but for now I'd recommend using our best judgment and generally avoiding a single line if statement without braces unless there's a great reason for it.

Reply:
done, the indentation is the one from eslint

Reply:
done, the indentation is the one from eslint